tests/integration/features/cleanup_int_test.go

```diff
@@ -58,7 +59,7 @@ var _ = Describe("feature cleanup", func() {
 			Expect(featuresHandler.Apply()).Should(Succeed())
 
 			// then
-			Eventually(createdSecretHasOwnerReferenceToOwningFeature(namespace, secretName)).
+			Eventually(createdSecretHasOwnerReferenceToOwningFeature(namespace)).
```

Comment:
Any reason to remove secret name as a param? With this change we hardcode the secret name in the func itself, not sure we want that.

Reply:
This was removed as linter complained as we were passing in the same secret name each time IIRC